source/release-notes/release-4-2-5.rst

```diff
@@ -53,11 +53,11 @@ Wazuh Kibana plugin
 Reference                                                         Description
 ==============================================================    =============
 `#3653 <https://github.com/wazuh/wazuh-kibana-app/pull/3653>`_    A compatibility issue between Wazuh 4.2 and the kibana 7.13.4 is now fixed. In addition, this fix is compatible with Kibana 7.10. 
-`#3654 <https://github.com/wazuh/wazuh-kibana-app/pull/3654>`_    Fixed error that produced the interactive screen of a new agent to break when selecting the windowsOS option. This fix is compatible with Kibana 7.10. 
-`#3668 <https://github.com/wazuh/wazuh-kibana-app/pull/3668>`_    This fix resolves a style compatibility issue of the breadcrumb navigation inside Kibana 7.14.2. 
-`#3670 <https://github.com/wazuh/wazuh-kibana-app/pull/3670>`_    This fix resolves Wazuh API token removal after logout with Kibana 7.13 and 7.14. 
+`#3654 <https://github.com/wazuh/wazuh-kibana-app/pull/3654>`_    This fix resolves an error that produced the interactive screen of a new agent to break when selecting the Windows OS option. This fix is compatible with Kibana 7.10. 
+`#3668 <https://github.com/wazuh/wazuh-kibana-app/pull/3668>`_    A style compatibility issue of the breadcrumb navigation inside Kibana 7.14.2 is now fixed. 
+`#3670 <https://github.com/wazuh/wazuh-kibana-app/pull/3670>`_    This fix resolves the Wazuh API token not being deprecated after logout with Kibana 7.13 and 7.14. 
 `#3672 <https://github.com/wazuh/wazuh-kibana-app/pull/3672>`_    A Group Configuration and Management Configuration error is now fixed when the user tries to go back after saving.
-`#3674 <https://github.com/wazuh/wazuh-kibana-app/pull/3674>`_    This fix resolves that the Panels and their titles are correctly displayed in the Welcome Overview and the Management Directory. In addition, it resolves that all the Wazuh menus are consistent without titles in gray. 
+`#3674 <https://github.com/wazuh/wazuh-kibana-app/pull/3674>`_    This fix resolves that the panels and their titles are correctly displayed in the Welcome Overview and the Management Directory. In addition, it resolves that all the Wazuh menus are consistent without titles in gray. 
```

Comment:
After accepting this suggestion, compile the branch and fix the table that will be broken due to the difference in the number of characters in this line. 

```suggestion
`#3674 <https://github.com/wazuh/wazuh-kibana-app/pull/3674>`_    With this fix, the panels and their titles are correctly displayed in the Welcome Overview and the Management Directory. In addition, it resolves that all the Wazuh menus are consistent without titles in gray. 
```